### DISCUSSION TITLE
Program Request (Unrelated to Shell)

### DISCUSSION THREAD

**Discussion by ghost** [moudey/Shell#136] [Ideas]
Hello,
I don't know if I have any right to ask for this or if this even is the right place to ask. But, could you try working on a program that implements Acrylic on parts of the OS such as titlebars, etc. (similar to how Mica for Everyone works)...

Something like "Acrylic for Everyone..."

Regards

**Comment by moudey** [comment]
Do you mean to add a Mica effect to Shell?
